src/components/projects-summary/projects-summary.controller.ts

```diff
@@ -2,34 +2,44 @@ import * as angular from 'angular';
 import * as _ from 'lodash';
 
 export class ProjectsSummaryController implements angular.IController {
-  static $inject = ['$element', '$scope', '$filter', 'ProjectsService', 'Logger', 'AuthService', 'DataService', 'Constants', 'AlertMessageService'];
+  static $inject = ['$element', '$rootScope', '$scope', '$filter', 'ProjectsService', 'Logger', 'AuthService',
```

Comment:
This list is getting long enough it's probably good to put one item on each line and sort them alphabetically (same for constructor below). I'm worried otherwise it will be too easy to get out of sync with the constructor args.